carma_wm_ctrl/src/WMBroadcaster.cpp

```diff
@@ -369,6 +374,115 @@ void WMBroadcaster::removeGeofence(std::shared_ptr<Geofence> gf_ptr)
   carma_wm::toBinMsg(send_data, &gf_msg_revert);
   map_update_pub_(gf_msg_revert);
 };
+  
+void  WMBroadcaster::routeCallbackMessage(const cav_msgs::Route& route_msg)
+{
+
+ cav_msgs::ControlRequest cR; 
+ cR =  routeCallbackMessageLogic(route_msg);
+ route_callmsg_pub_(cR);
+
+}
+
+
+cav_msgs::ControlRequest WMBroadcaster::routeCallbackMessageLogic(const cav_msgs::Route& route_msg) 
+{
+
+
+  auto path = lanelet::ConstLanelets(); 
+
+
+  if (!current_map_) {
+// Return / log warning etc.
+  ROS_INFO_STREAM("Value 'current_map_' does not exist.");
```

Comment:
use ROS_WARN_STREAM here as this should be a warning that we got a route but don't have a map. Also list what function the message is coming from in the error message.